config.json

```diff
@@ -41,6 +41,17 @@
         "strings"
       ]
     },
+    {
+      "slug": "darts",
+      "uuid": "4d400a44-b190-4a0c-affb-99fad8ea18da",
+      "core": false,
+      "unlocked_by": "two-fer",
+      "difficulty": 1,
+      "topics": [
+        "games",
+        "math"
```

Comment:
According to the [topics list](https://github.com/exercism/problem-specifications/blob/master/TOPICS.txt) I think this should be `mathematics` 🙂

Reply:
According to #1499 this was changed from 'mathematics' to 'math' in August and 'math' is used throughout the config file. So I would keep it if there have not been any new changes since then and suggest to adapt the topics list accordingly. I could open an issue on the resp. rep after checking there about this decision.

Reply:
Oh, I didn't see this list of exercises that should have math topics. I will check for it in the problem-spec repo. 
Of course I would then remove the math topic from this new exercise. The darts exercise was added just a few days ago to the problem-spec repo.

Reply:
I see, I didn't realise it was so new 🙂 I don't think there's a definite list, there was just a list they made at the time of raising https://github.com/exercism/exercism/issues/4110. So from my understanding of that we can add maths as a topic to new exercises but we should avoid doing so unless maths is really important to the exercise as it might put people off 🙂